### DISCUSSION TITLE
Linux KeePassXC Location and Linux Uninstall

### DISCUSSION THREAD

**Discussion by savidudu** [keepassxreboot/keepassxc#8997] [Q&A] [answered]
Hello Team,
 
 I would like to know what all options available to install KeePassXC on Linux box apart from AppImage ?
 
 Also, If we are using KeePassXC through AppImage then in which directory (var, op, etc) will be having the files, I tried checking but no luck.
 
 Apart from AppImage what all option I can use to install the KeePassXC ?
 
 Also, How to uninstall the KeePassXC from the Linux machine after installation.
 
 Thanks,

**Comment by droidmonkey** [comment, marked as answer]
https://keepassxc.org/download

You'll have to figure out the install/uninstall part based on your Linux distro.
